sensor/common/scan/scan.go

```diff
@@ -61,7 +62,9 @@ type LocalScan struct {
 	getGlobalRegistry         func(*storage.ImageName) (registryTypes.ImageRegistry, error)
 
 	// scanSemaphore limits the number of active scans.
-	scanSemaphore        *semaphore.Weighted
+	scanSemaphore *semaphore.Weighted
+	// adHocScanSemaphore limits the number of delegated scans.
+	adHocScanSemaphore   *semaphore.Weighted
```

Comment:
how come we change the terminology here?

Reply:
I was hesitating about the proper name.